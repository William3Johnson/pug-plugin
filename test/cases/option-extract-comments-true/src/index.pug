html
  head
    script(src=require('./vendor.js'))
  body