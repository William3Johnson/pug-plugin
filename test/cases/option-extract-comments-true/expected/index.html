<!DOCTYPE html><html><head><script src="/assets/js/vendor.04906076.js"></script></head><body></body></html>